bashrc

```diff
@@ -44,8 +44,8 @@ export SAVEHIST=100
 # Fichier ou est stocké l'historique :
 export HISTFILE=$HOME/.history
 HISTIGNORE="clear:bg:fg:cd:cd -:exit:date:w:* --help"
-HISTCONTROL="ignorespace"
-HISTCONTROL="ignoredups"
+#HISTCONTROL="ignorespace"
+#HISTCONTROL="ignoredups"
```

Comment:
These are redundant. The `ignoreboth` option implies both `ignorespace` and `ignoredups`, as described in the `bash` man page.